File: analysis/source/prepare_data/create_data.r
library(yaml)
CONFIG <- yaml.load_file("config_global.yaml")
build_path <- CONFIG$build$prepare_data


main <- function() {
    x <- 1:300000
    write.table(x, sprintf("%s/data.txt", build_path),
                row.names = FALSE, col.names = TRUE, quote = FALSE)
}
main()

File: analysis/source/prepare_data/create_data.py
import yaml


def main():
    CONFIG = yaml.load(open('config_global.yaml', 'rU'))
    with open('%s/data.txt' % CONFIG['build']['prepare_data'], 'wb') as f:
        f.write('x\n')
        f.writelines(['%s\n' % x for x in range(1, 300001)])

main()
